include/ozo/type_traits.h

```diff
@@ -92,11 +113,53 @@ inline auto is_null(const ::std::weak_ptr<T>& v) noexcept {
 }
 
 template <typename T>
-constexpr auto is_null(const T&) noexcept ->
-    std::enable_if_t<!is_nullable<std::decay_t<T>>::value, std::false_type> {
+constexpr auto is_null(const T&) noexcept -> std::enable_if_t<!Nullable<T>, std::false_type> {
     return {};
 }
 
+template <typename T>
+void init_nullable(T&) {}
+
+template <typename T>
+void init_nullable(::boost::optional<T>& o) {
+    o.emplace();
```

Comment:
Is it really needed to do emplace for already initialized optional?

Reply:
no, fixed